-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 55.60 s <<< FAILURE! -- in TestSuite
com.luma.base.TestRunner.runScenario["", "login"](4) -- Time elapsed: 11.57 s <<< FAILURE!
java.lang.AssertionError: expected [ssss] but found []
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:629)
	at org.testng.Assert.assertEquals(Assert.java:639)
	at com.luma.stepdefinition.A0_lumaLoginStepDefinition.loggedMsg(A0_lumaLoginStepDefinition.java:51)
	at âœ½.getValidCredentialMsg(file:///D:/01%20Sameer/01%20Java/01%20Project/ssWorkspace/Z85_LUMA/src/test/resources/Features/Login.feature:19)

